performance-test/tasks/notification_tasks.py

```
@@ -1,15 +1,54 @@
+                try:
+                    data = response.json()
+                    unread_list = data.get("content", [])
+
+                    if unread_list:
+                        action = random.random()
+                        if action < 0.7: # 전체 확인(대부분)
+                            self.client.patch("/api/notifications",
+                                              headers=headers,
+                                              name="/api/notifications (All)")
+                        else: # 개별 확인
+                            target_id = random.choice(unread_list).get("id")
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)